pypodio2/encode.py

```diff
@@ -39,22 +39,22 @@ def gen_boundary():
 
 
 def encode_and_quote(data):
-    """If ``data`` is unicode, return urllib.quote_plus(data.encode("utf-8"))
-    otherwise return urllib.quote_plus(data)"""
+    """If ``data`` is unicode, return urllib.parse.quote_plus(data.encode("utf-8"))
+    otherwise return urllib.parse.quote_plus(data)"""
     if data is None:
         return None
 
-    if isinstance(data, unicode):
+    if isinstance(data, str):
```

Comment:
In Python3, the `unicode` type is renamed `str` (and `str` is renamed `bytes`, but we didn't use `str` anywhere).